recipe/meta.yaml

```diff
@@ -5066,6 +5066,24 @@ outputs:
       summary: |
         Implementation of Kerberos V5 libraries
 
+  - name: posix
+    version: 2.0.0.20230914
+    build:
+      noarch: generic
+    requirements:
+      run:
+        - m2-conda-epoch 20230914
+        - m2-base
+        - m2-make
+        - m2-tar
+        - m2-zip
+        - m2-unzip
+        - m2-diffutils
+    about:
+      home: https://github.com/conda-forge/m2-recipes-feedstock
```

Comment:
Q: Is this a private repo? I can't access it.

Reply:
No, that's a typo for this repo.